src/main/java/DukeException.java

```diff
@@ -0,0 +1,5 @@
+public class DukeException extends Exception{
```

Comment:
For this class and your Task class, can you indent it with 4 spaces instead of 2?